tests/CMakeLists.txt

```diff
@@ -1,12 +1,17 @@
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
 
 find_package(Catch2 3.1.0 REQUIRED)
+find_package(range-v3 REQUIRED)
```

Comment:
I would prefer not to because I can't specify what version we are using in the pacakge.xml and RHEL and Ubuntu22 might have different versions.

Reply:
You can specify a minimum version that both meet so long as the major versions are the same. If one provides 3.1 and the other 3.2, you can write
```cmake
find_package(ranges-v3 3.1 REQUIRED)
```
So that we're sure that someone at least won't use a version that is too low to satisfy our requirements.

Reply:
Done, hopefully, that works. I believe 0.11.0 is the minimum of the two versions I want to support.